unattended_scripts/install_functions/opendistro/wazuh-cert-tool.sh

```diff
@@ -283,72 +288,78 @@ main() {
     if [ "$EUID" -ne 0 ]; then
         logger_cert -e "This script must be run as root."
         exit 1;
-    fi    
+    fi
 
-    if [ -n "$1" ]; then      
+    if [ -n "$1" ]; then
         while [ -n "$1" ]
         do
-            case "$1" in 
-            "-a"|"--admin-certificates") 
+            case "$1" in
+            "-a"|"--admin-certificates")
                 cadmin=1
                 shift 1
-                ;;     
-            "-ca"|"--root-ca-certificate") 
+                ;;
+            "-ca"|"--root-ca-certificate")
                 ca=1
                 shift 1
-                ;;                           
-            "-e"|"--elasticsearch-certificates") 
+                ;;
+            "-e"|"--elasticsearch-certificates")
                 celasticsearch=1
                 shift 1
-                ;; 
-            "-w"|"--wazuh-certificates") 
+                ;;
+            "-w"|"--wazuh-certificates")
                 cwazuh=1
                 shift 1
-                ;;   
-            "-k"|"--kibana-certificates") 
+                ;;
+            "-k"|"--kibana-certificates")
                 ckibana=1
                 shift 1
-                ;;                               
-            "-v"|"--verbose") 
-                debugEnabled=1          
+                ;;
+            "-v"|"--verbose")
+                debugEnabled=1
                 shift 1
-                ;;                                 
-            "-h"|"--help")        
+                ;;
+            "-h"|"--help")
                 getHelp
-                ;;                                         
+                ;;
             *)
                 getHelp
             esac
-        done    
+        done
+
+        checkOpenSSL
+
+        if [ -z "$openssl_installed" ]; then
+            logger_cert -e "OpenSSL isn't installed in this system. Install it to continue the creation of certificates"
+        fi
```

Comment:
If it is not installed, install the package, this can be done in the same OpenSSL check, if it is installed, the system will ignore it.